docs/index.md

```diff
@@ -35,59 +35,42 @@ features:
 
 Welcome to all new visitors!
 
-This project is still very new and the team is still working hard on a number
-of areas of the whole project. That includes the website, package repository and
-GitHub organisation as well as the editor itself.
-
-The below is just some items that we are currently working on that might answer
-any questions you have at this early stage of the project.
-
 ### Downloads and Releases
 
 We are constantly working on the editor and maintain both a rolling release and regular release model for our binaries.
 You can find out more about the difference and download a binary of Pulsar by following [this link](https://pulsar-edit.dev/download.html)
 and following the instructions.
 
-We are looking to support distro repositories such as Debian/Ubuntu repos,
-Fedora, Snap, Flatpak, Arch/AUR etc. but we need to get to a good point with
-a stable release before we instigate that so for now the downloadable binaries
-are the main supported downloads.
+There are some community-maintained Pulsar packages in various formats, such as Flatpak, AUR, etc. The effort put into maintaining these packages is very much appreciated!
```

Comment:
```suggestion
There are some community-maintained packaged releases of Pulsar in various formats, such as Flatpak, AUR, etc. The effort put into maintaining these packages is very much appreciated!
```
Purely optional (and very happy for further suggestions) - yes the suggestion is a lot more unweildy but it makes it distinct from "Pulsar packages" i.e. the editor plugins.

Reply:
The whole paragraph might need a little second-time-over regarding "package" this and "package" that.

A good start, though.

Reply:
Agree it is a bit hard going, probably does need a bit more rework.

Reply:
I have a gut feeling the answer is probably "say less" in this paragraph. So I think a trim will do it good once I have fresh eyes, or if enyone else wants to take a crack at just not saying as much, or staying to just. some facts and light remarks rather than being comprehensive, maybe. ASince it's just an infobox, after all.

Hey, maybe the cure is to get more into the weeds on Downloads in terms of _acknowledging_ these other formats, even if we don't tier-1 support them (and even if we say we don't totally support them), and hint that users interested in these can see more on the _Downloads page_?? Seems like a a good idea to me.